### PR TITLE
Tk 19

### PR DESCRIPTION
**Pull request by johangv**

### CODE REVIEW

File: src/pages/Home/Home.jsx [hunk 1]
```diff
@@ -22,6 +25,7 @@ const Home = () => {
           <PokeList /> 
         </div> }
     </div>
```
Comment (davidlopez9926) [inline src/pages/Home/Home.jsx]: identation

File: src/App.js [hunk 2]
```diff
@@ -1,20 +1,31 @@
...
-          <Home />
-        </div>
-      </PokedexLayout>
+        <PokedexLayout>
+          <div className="App bg" >
+            <Router>
+              <Routes>
+                <Route path="/" element={<Navigate to='/home'/>}/>
+                <Route path="*" element={<Navigate to='/home'/>}/>
+                <Route path="home" element={<Home />} />
```
Comment (davidlopez9926) [inline src/App.js]: put the /

**Review by bruno9926 — CHANGES_REQUESTED** [review]
do the changes

File: src/App.js (hunk 2 above)
Comment (johangv) [reply src/App.js]: Fixed

File: src/pages/Home/Home.jsx (hunk 1 above)
Comment (johangv) [reply src/pages/Home/Home.jsx]: Fixed

**Review by bruno9926 — APPROVED** [review]
All ok ;)

**Pull request merged**